Multisig/UI/WalletConnectV2/WalletConnectManager.swift

```diff
@@ -15,90 +15,98 @@ import WalletConnectUtils
-            .sink { [unowned self] sessions in
-                sessions.forEach { session in
-                    session.namespaces.forEach {
-                        $0.value.accounts.forEach {
-                            if let safe = Safe.by(address: $0.address, chainId: $0.reference) {
-                                safe.addSession(topic: session.topic)
+                .receive(on: DispatchQueue.main)
+                .sink { [unowned self] sessions in
+                    sessions.forEach { session in
+                        session.namespaces.forEach {
+                            $0.value.accounts.forEach {
+                                if let safe = Safe.by(address: $0.address, chainId: $0.reference) {
+                                    safe.addSession(topic: session.topic)
+                                }
                             }
                         }
-                    }
 
-                    if let dappConnectedTrackingEvent = dappConnectedTrackingEvent {
-                        // parameter names should not exceed 100 chars
-                        let dappName = session.peer.name.prefix(100)
-                        Tracker.trackEvent(dappConnectedTrackingEvent, parameters: ["dapp_name" : dappName])
+                        if let dappConnectedTrackingEvent = dappConnectedTrackingEvent, !session.peer.name.isEmpty {
+                            let dappName = session.peer.name.prefix(100)
+                            Tracker.trackEvent(dappConnectedTrackingEvent, parameters: ["dapp_name": dappName])
+                        }
                     }
-                }
 
-                dappConnectedTrackingEvent = nil
-                NotificationCenter.default.post(name: .wcDidConnectSafeServer, object: self)
-            }.store(in: &publishers)
+                    dappConnectedTrackingEvent = nil
+                    NotificationCenter.default.post(name: .wcDidConnectSafeServer, object: self)
+                }
+                .store(in: &publishers)
 
         Sign.instance.sessionDeletePublisher
-            .receive(on: DispatchQueue.main)
-            .sink { [unowned self] response in
-                deleteStoredSession(topic: response.0)
-                NotificationCenter.default.post(name: .wcDidDisconnectSafeServer, object: self)
-            }.store(in: &publishers)
+                .receive(on: DispatchQueue.main)
+                .sink { [unowned self] response in
+                    deleteStoredSession(topic: response.0)
+                    NotificationCenter.default.post(name: .wcDidDisconnectSafeServer, object: self)
+                }
+                .store(in: &publishers)
     }
 
     func canConnect(url: String) -> Bool {
         WalletConnectURI(string: url) != nil
     }
 
     func pairClient(url: String, trackingEvent: TrackingEvent?) {
-        guard let uri = WalletConnectURI(string: url) else { return }
+        guard let uri = WalletConnectURI(string: url) else {
```

Comment:
Based  our coding style, this return should be on same line, so as the original line